### PR TITLE
fix(KtTable-selected-expandMultiple): fixed selected binding

### PR DESCRIPTION
**Pull request by carsoli**
#128

### CODE REVIEW

File: packages/kotti-table/src/logic/select.js [hunk 1]
```diff
@@ -61,14 +61,22 @@ export const getters = {
 			: row[state.rowKey]
 	},
 	isSelected(state, row) {
-		return (state.selection || []).indexOf(row) > -1
+		return (
+			(state.selection.map(e => JSON.stringify(e)) || []).indexOf(
```
Comment (FlorianWendelborn) [inline packages/kotti-table/src/logic/select.js]: https://www.npmjs.com/package/deep-eql or https://www.npmjs.com/package/shallow-equal

**Comment by amrdraz** [comment]
@carsoli testing on netlify the binding preview stop working after I select all, which doesn't happen with the v-model mode 
also there's a merge conflict

**Comment by carsoli** [comment]
> @carsoli testing on netlify the binding preview stop working after I select all, which doesn't happen with the v-model mode
Great Catch! Will see to it in a separate to commit.

> also there's a merge conflict
Resolved, now. 
@amrdraz

File: packages/kotti-table/src/logic/select.js (hunk 1 above)
Comment (amrdraz) [reply packages/kotti-table/src/logic/select.js]: @carsoli did you address this?

@FlorianWendelborn would deep equal windup interacting with Vue's proxies?
I think stringily does the job of deep equal as far as the kind of props used in the columns are concerned as far as we need it too, if we're talking performance we may be ok with shallow equal

**Pull request merged**